spring-integration-jpa/src/main/java/org/springframework/integration/jpa/outbound/JpaOutboundGatewayFactoryBean.java

```diff
@@ -38,6 +39,7 @@
  */
 public class JpaOutboundGatewayFactoryBean extends AbstractSimpleMessageHandlerFactoryBean<JpaOutboundGateway> {
 
+	@SuppressWarnings("NullAway") // will be initialized in xml parser
```

Comment:
We cannot assume how this class is going to be used. So, make it nullable and assert in the `createHandler()`